src/main/java/ezvcard/io/json/JCardModule.java

```diff
@@ -0,0 +1,135 @@
+package ezvcard.io.json;
+
+import static java.lang.Integer.parseInt;
+
+import com.fasterxml.jackson.core.Version;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+
+import ezvcard.Ezvcard;
+import ezvcard.VCard;
+import ezvcard.io.scribe.ScribeIndex;
+import ezvcard.io.scribe.VCardPropertyScribe;
+import ezvcard.property.VCardProperty;
+
+/**
+ * Module for jackson-databind with JCardSerializer and JCardDeserializer as the
+ * default serializers for {@link VCard}s
+ * 
+ * @author buddy
+ *
+ */
+public class JCardModule extends SimpleModule {
+	private static final long serialVersionUID = 1L;
+
+	public static final String MODULE_NAME = "JCardModule";
+	private static final String[] V = Ezvcard.VERSION.split("[.-]");
+	public static final Version MODULE_VERSION = new Version(parseInt(V[0]), parseInt(V[1]), parseInt(V[2]),
+			V.length > 3 ? V[3] : "RELEASE", "com.googlecode.ez-vcard", "ez-vcard");
+
+	private final JCardDeserializer deserializer = new JCardDeserializer();
+	private final JCardSerializer serializer = new JCardSerializer();
+
+	/**
+	 * Creates a JCardModule, which can be registered with an ObjectMapper using
+	 * {@link ObjectMapper#registerModule}
+	 */
+	public JCardModule() {
+		super(MODULE_NAME, MODULE_VERSION);
+
+		// Register the same scribe with both, as per the documentation of
+		// this.getScribeIndex()
+		ScribeIndex index = new ScribeIndex();
```

Comment:
Because the serializer and deserializer are using the same `ScribeIndex` instance, this should be stored in a class field.  Then, the `registerScribe` method can add the scribe to this `ScribeIndex` instance (it doesn't have to call `registerScribe` on the serializer and deserializer).  The `getScribeIndex` would obviously return it.  And `setScribeIndex` would set it, along calling the `setScribeIndex` methods on the serializer and deserializer.  Does that make sense?